README.md

```diff
@@ -72,3 +72,4 @@ This section has moved here: [https://facebook.github.io/create-react-app/docs/t
 ## Contributors
 
 Christian Bingman - [@ChristianBingman](https://github.com/ChristianBingman)
+Connor Hunter - [@Mohpann](https://github.com/Mohpann)
```

Comment:
```suggestion
Connor Hunter - [@Mohpann](https://github.com/Mohpann) 
resolves #29 
```

Reply:
This should be in the commit message, not on the README